demo/mpfaof90.F90

```diff
@@ -0,0 +1,225 @@
+module f90module
```

Comment:
This is great and I hope goes a long way to help people see that this could work trivially in fortran codes